File: sdk/dart/lib/model/price_schedule.dart
part of swagger.api;


@Entity()
class PriceSchedule {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'Name')
  String name = null;
  

  @Property(name: 'ApplyTax')
  bool applyTax = null;
  

  @Property(name: 'ApplyShipping')
  bool applyShipping = null;
  

  @Property(name: 'MaxQuantity')
  int maxQuantity = null;
  

  @Property(name: 'UseCumulativeQuantity')
  bool useCumulativeQuantity = null;
  

  @Property(name: 'RestrictedQuantity')
  bool restrictedQuantity = null;
  

  @Property(name: 'OrderType')
  String orderType = null;
  

  @Property(name: 'PriceBreaks')
  List<ShipmentItem> priceBreaks = [];
  

  @Property(name: 'xp')
  Object xp = null;
  
  PriceSchedule();

  @override
  String toString()  {
    return 'PriceSchedule[ID=$ID, name=$name, applyTax=$applyTax, applyShipping=$applyShipping, maxQuantity=$maxQuantity, useCumulativeQuantity=$useCumulativeQuantity, restrictedQuantity=$restrictedQuantity, orderType=$orderType, priceBreaks=$priceBreaks, xp=$xp, ]';
  }

}

File: sdk/javascript-closure-angular/API/Client/PriceSchedule.js
goog.provide('API.Client.PriceSchedule');

/**
 * @record
 */
API.Client.PriceSchedule = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.PriceSchedule.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.PriceSchedule.prototype.name;

/**
 * @type {!boolean}
 * @export
 */
API.Client.PriceSchedule.prototype.applyTax;

/**
 * @type {!boolean}
 * @export
 */
API.Client.PriceSchedule.prototype.applyShipping;

/**
 * @type {!number}
 * @export
 */
API.Client.PriceSchedule.prototype.maxQuantity;

/**
 * @type {!boolean}
 * @export
 */
API.Client.PriceSchedule.prototype.useCumulativeQuantity;

/**
 * @type {!boolean}
 * @export
 */
API.Client.PriceSchedule.prototype.restrictedQuantity;

/**
 * @type {!string}
 * @export
 */
API.Client.PriceSchedule.prototype.orderType;

/**
 * @type {!Array<!API.Client.ShipmentItem>}
 * @export
 */
API.Client.PriceSchedule.prototype.priceBreaks;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.PriceSchedule.prototype.xp;

